### DISCUSSION TITLE
Docker unable to built - no URLs in mirrorlist

### DISCUSSION THREAD

**Discussion by jackty9** [Azure/azure-iot-sdk-c#2264] [Q&A] [answered]
Hi all,
I am following this [tutorial ](https://docs.microsoft.com/en-us/azure/iot-edge/tutorial-machine-learning-edge-05-configure-edge-device?view=iotedge-2020-11) here to configure a transparent gateway. I have a VM setup via Azure and tried to build a docker image but it failed with the following message. I tried to Google and Stackoverflow but couldn't find any concrete solution. Did anyone follow this tutorial and face the same problem? 

![image](https://user-images.githubusercontent.com/36817589/159046641-349dbc54-4be8-4667-9fca-3551ee27e565.png)

**Comment by ericwolz** [comment, marked as answer]
[CentOS Linux 8 had reached the End Of Life (EOL)](https://www.centos.org/centos-linux-eol/) on December 31st, 2021. It means that CentOS 8 will no longer receive development resources from the official CentOS project. After Dec 31st, 2021, if you need to update your CentOS, you need to change the mirrors to [vault.centos.org](https://vault.centos.org/) where they will be archived permanently.

https://techglimpse.com/failed-metadata-repo-appstream-centos-8/
